pkg/controller/podautoscaler/horizontal_test.go

```diff
@@ -116,17 +117,17 @@ func (tc *testCase) prepareTestClient(t *testing.T) *fake.Clientset {
 	})
 
 	fakeClient.AddReactor("get", "replicationController", func(action core.Action) (handled bool, ret runtime.Object, err error) {
-		obj := &extensions.Scale{
+		obj := &autoscaling.Scale{
```

Comment:
We should not be changing the type of replicationController's scale subresource.